python-package/fairgbm/sklearn.py

```diff
@@ -1033,3 +1199,51 @@ def fit(self, X, y,
                    "eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n"
                    f"{' ':12}The evaluation positions of the specified metric.\n"
                    f"{' ':8}{_early_stop}{_after_early_stop}")
+
+
+class FairGBMClassifier(LGBMClassifier):
+    """Helper wrapper to use FairGBM with an sklearn-like API."""
+
+    FAIRGBM_OBJECTIVE = "constrained_cross_entropy"
+
+    def __init__(self, boosting_type='gbdt', num_leaves=31, 
+                 max_depth=-1, learning_rate=0.1, n_estimators=100, 
+                 subsample_for_bin=200000, class_weight=None, 
+                 min_split_gain=0, min_child_weight=0.001, min_child_samples=20, 
+                 subsample=1, subsample_freq=0, colsample_bytree=1, 
+                 reg_alpha=0, reg_lambda=0, random_state=None, 
+                 n_jobs=-1, silent=True, importance_type='split',
+                 multipler_learning_rate=50_000, constraint_type="FPR", 
+                 global_constraint_type="FPR, FNR", global_target_fpr=0.05, 
```

Comment:
Should we set these defaults for the open-source package? @saleiro 

I think a lot of people will use the default values and these are somewhat specific to our projects.

Reply:
Moved to #31